code/API_definitions/number_verification.yaml

```diff
@@ -3,6 +3,9 @@ info:
   title: Number Verification
   description: |
     Service Enabling Network Function API to verify that the provided **mobile phone number** is the one used in the device. It verifies that the user is using a device with the same *mobile phone number* as it is declared.
+
+    This API derives from the GSMA Mobile Connect Account Takeover Protection specification [Mobile Connect Number Verify](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf).  For more about Mobile Connect, please see [about Mobile Connect](https://mobileconnect.io/).
```

Comment:
Yaml should not include this info. This info should be in the .md files within API documentation. Yaml is API spec and has to stick to API Definition

Reply:
You're right @bigludo7, I didn't realize that. Suggestion then is to include the reference in the .yaml plus remove rest of documentation, meeting commonalities agreement